src/app/home/home.component.html

```diff
@@ -13,7 +13,7 @@
     (click)="navigate('chat')"
     (tap)="navigate('chat')"
   >
-    CHAT
+    CHAT {{ !ownSession && newMessages > 0 && currentTab !== 'chat' ? '(' + newMessages + ')' : '' }}
```

Comment:
Should be fixed. The default span style for `vertical-align` was somehow overwritten